linalg/sundials.cpp

```diff
@@ -367,14 +497,15 @@ N_Vector SundialsNVector::MakeNVector(bool use_device)
 #ifdef MFEM_USE_CUDA
    if (use_device)
    {
-      x = N_VNewWithMemHelp_Cuda(0, UseManagedMemory(), sunmemHelper);
+      x = N_VNewWithMemHelp_Cuda(0, UseManagedMemory(), sunmemHelper,
```

Comment:
Should `sunmemHelper` be `Sundials::GetMemHelper()`?

Reply:
Also, need a `SUNContext` argument at the end.

Reply:
Should all be addressed!